data/actions/projectRouter.js

```diff
@@ -0,0 +1,85 @@
+const express = require("express");
+const db = require("../helpers/projectModel");
+const router = express.Router();
+router.use(express.json());
```

Comment:
you can use a server.use(express.json()) in your server.js file so that you dont need to add this to every router